apps/web/components/ui/UsernameAvailability/UsernameTextfield.tsx

```diff
@@ -135,8 +135,8 @@ const UsernameTextfield = (props: ICustomUsernameProps & Partial<React.Component
             {...rest}
           />
           {currentUsername !== inputUsernameValue && (
-            <div className="absolute right-[2px] top-6 flex flex-row">
-              <span className={classNames("mx-2 py-3.5")}>
+            <div className="absolute right-[2px] top-6 flex h-7 flex-row">
+              <span className={classNames("mx-0 bg-white p-3")}>
```

Comment:
I dont think we can use bg-white here as we need this to work in dark mode also. Off the top of my head `bg-default` should work. 

Thanks

Reply:
Actually I applied dark mode with chrome extension it is working.